xscen/catalog.py

```diff
@@ -960,7 +960,7 @@ def subset_file_coverage(
                 intervals[files_in_range].map(
                     lambda x: min(x.right, period_interval.right)
                 ),
-            ).length
+            ).length.sum()
```

Comment:
Have you tested it? I don't see a `.sum()` in the list of functions.

Reply:
Ah indeed, I think that the output type of `IntervalIndex.length` is just not documented: https://pandas.pydata.org/docs/reference/api/pandas.IntervalIndex.length.html